.travis.yml

```diff
@@ -1,14 +1,19 @@
 language: go
 
+go:
+  - "1.11.x"
+  - "1.12beta2"
+
+os:
+  - linux
+  - osx
+  - windows
```

Comment:
just a nit - do you really need to test two Go versions on all OSs? I generally test many Go versions on linux container, and only the latest stable go version on other OSs. This makes CI runs much faster, particularly because non-linux builds tend to be VMs instead of containers, making them considerably slower to start up.

It's also kinda unlikely IMO that something would break on Go 1.11 on Windows, but not on Go 1.11 on Linux. So unless you have tons of OS-specific code, I personally think this is overkill.

Reply:
I think this is fair comment. The only reason I am running on the full matrix is that if you end up having a version-specific output check, e.g. via `stdout`, then it _does_ make sense to check that on all OS-es. Because it's all too easy to write a pattern that only works on Unix. 

On the basis we're not making tonnes of changes for now, I'll leave as is (for now).